src/main/kotlin/no/digdir/accessrequestapi/controller/HandlekurvController.kt

```diff
@@ -0,0 +1,40 @@
+package no.digdir.accessrequestapi.controller
+
+import no.digdir.accessrequestapi.client.FelleskatalogClient
+import no.digdir.accessrequestapi.configuration.FdkUrls
+import no.digdir.accessrequestapi.model.DatasetLanguage
+import no.digdir.accessrequestapi.model.Handlekurv
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import java.util.UUID
+
+@RestController
+@RequestMapping(value = ["/handlekurv"], produces = ["application/json"])
```

Comment:
vi prøver å holde oss borte fra norsk i koden og `handlekurv` vil uansett bare være treffende for de som skal til den løsningen, det kan jo potensielt komme requests hit som skal andre steder. Hva med `access-request`?

og dermed AccessRequestController, getAccessRequest osv